services/plugins/plugins/schemas.go

```diff
@@ -65,22 +65,48 @@ var pluginSchema = map[string]*schema.Schema{
 	"created_at": {
 		Type:        schema.TypeString,
 		Computed:    true,
-		Description: "When it was created",
+		Description: "When the plugin was created",
 	},
 	"created_by": {
 		Type:        schema.TypeString,
 		Computed:    true,
-		Description: "Who created it",
+		Description: "Who created the plugin",
+	},
+	"last_modified_by": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Who modified the plugin the last",
 	},
 	"last_modified_at": {
 		Type:        schema.TypeString,
 		Computed:    true,
-		Description: "When it was last modified",
+		Description: "When the plugin was last modified",
 	},
-	"last_modified_by": {
+	"function_name": {
```

Comment:
I don't think we expose nor that we should expose it

Reply:
Ok I will delete this in the next commit.